### DISCUSSION TITLE
Tags with mini.surround

### DISCUSSION THREAD

**Discussion by Caddy121** [nvim-mini/mini.nvim#207] [Q&A] [answered]
How do I surround a line with, for example, an HTML <p> tag?

**Comment by echasnovski** [comment, marked as answer]
Try `sa_t`, type tag, press `Enter`.

**Reply by Caddy121** [reply]
That works! Thank you very much!

**Comment by kaung-minkhant** [comment]
Is there a way to support completions for the tag input?

**Reply by echasnovski** [reply]
I am afraid, not at the moment. There is no completion possible when getting text from a user with `vim.fn.input()`. Maybe if/when there is an organized transition across all modules to use customizable `vim.ui.input()`, but that is not top priority right now.
